internal/constants/status/order.go

```diff
@@ -0,0 +1,6 @@
+package status
+
+const NewID = 1
+const ProcessingID = 2
+const InvalidID = 3
+const ProcessedID = 4
```

Comment:
```suggestion
const (
    NewID = iota + 1
    ProcessingID
    InvalidID
    ProcessedID
)
```
Магия https://go.dev/play/p/vb_H0e7upb0

Reply:
поправил